### DISCUSSION TITLE
2024/06/01/from-eslint-and-prettier-to-biome/

### DISCUSSION THREAD

**Discussion by giscus** [KittyGiraudel/site#164] [Announcements]
# 2024/06/01/from-eslint-and-prettier-to-biome/

I’m Kitty Giraudel, a non-binary trans frontend developer based in Berlin, focused on accessibility and inclusivity.

https://kittygiraudel.com/2024/06/01/from-eslint-and-prettier-to-biome/

**Comment by james-s-w-clark** [comment]
Thanks for sharing - this is really helpful!

**Comment by bheemreddy-samsara** [comment]
Any benchmarks you could share ?
